src/pages/rt-fees-schedule/forms/FeesDetailsForm.js

```diff
@@ -0,0 +1,156 @@
+import { Grid, Box, Checkbox } from '@mui/material'
```

Comment:
remove
import { Grid, Box, Checkbox } from '@mui/material'
ResourceIds
useState
yup
they are not used